### DISCUSSION TITLE
Search recently activated users

### DISCUSSION THREAD

**Discussion by vkaelin** [zitadel/zitadel#7447] [Q&A] [answered]
Hello 👋

In our application, we need to retrieve users who have recently activated their accounts (for example, within the last 24 hours).

I've been reviewing the documentation (https://zitadel.com/docs/apis/resources/user_service/user-service-list-users) but I can't seem to find a method to sort by this attribute using the `sortingColumn` parameter.

In addition, is it possible to determine when a user is activated?

Is there another approach I'm overlooking, or is this currently not feasible?

Thank you very much!

**Comment by fforootd** [comment, marked as answer]
Hey, good question.

We have an event-api that can be queried for such infos.
Checkout this example here https://zitadel.com/docs/guides/integrate/event-api#example-get-user-changes-since-a-specific-date

There is also a UI under /ui/console/instance?id=events that uses this API

**Reply by vkaelin** [reply]
Thanks a lot for the quick answer!

Just to be sure, is it possible to access the event-api with a service user? Because in my application when I try to use my Org Owner service user, I get a 403 on the event list endpoint.

`403 Forbidden from POST https://<CUSTOM_DOMAIN_HIDDEN_HERE>/admin/v1/events/types/_search`

**Reply by hifabienne** [reply]
Yes it is, but you will need an IAM OWNER, as this is in the admin api

**Reply by vkaelin** [reply]
Thanks a lot to both of you!
